src/main/webapp/app/course/plagiarism-cases/student-view/detail-view/plagiarism-case-student-detail-view.component.html

```diff
@@ -46,12 +46,24 @@ <h3>{{ 'artemisApp.plagiarism.plagiarismCases.conversation' | artemisTranslate }
                             <jhi-post #post [readOnlyMode]="false" [posting]="posts[0]" [showAnswers]="true" [isCourseMessagesPage]="false" [hasChannelModerationRights]="false">
                             </jhi-post>
                         </div>
-                        <div class="alert alert-warning my-2" *ngIf="!this.plagiarismCase.createdByContinuousPlagiarismControl && (this.posts[0].answers?.length ?? 0) === 0">
+                        <div
+                            class="alert alert-warning my-2"
+                            *ngIf="
+                                (this.posts[0].answers?.length ?? 0) === 0 &&
+                                (!this.plagiarismCase.createdByContinuousPlagiarismControl || (this.plagiarismCase.createdByContinuousPlagiarismControl && this.isAfterDueDate))
+                            "
+                        >
                             <h5
                                 *ngIf="posts[0].creationDate"
                                 [innerHTML]="
                                     'artemisApp.plagiarism.replyToAccusationDueDateNotification'
-                                        | artemisTranslate: { dueDate: posts[0].creationDate.add(7, 'day').format('DD.MM.YYYY') }
+                                        | artemisTranslate
+                                            : {
+                                                  dueDate: (plagiarismCase.createdByContinuousPlagiarismControl
+                                                      ? dayjs(plagiarismCase.exercise!.dueDate!).add(7, 'day')
+                                                      : posts[0].creationDate.add(7, 'day')
```

Comment:
This timeframe needs to be adjustable by an instructor. If the due date is right before Christmas, you cannot expect  students to respond within 7 days.
With the posts it was not such a big problem, since the posts could be created on dates that do not leave to such a problem.

Reply:
Thanks for your suggestion. I've added as an option to configure per exercise (screenshot attached to the description)

Reply:
Shouldn't you also use the newly added variable in the server to check if a student submitted the response within the correct timeframe?
And the UI also does not change after the time for a response is over, does it?

Reply:
Response submission is not block and  UI does not change after the deadline. This is the behaviour of the original feature currently deployed on production.

I suggest we make an issue about that and implement the improvement in separate PR, so that the cpc project keeps going. What do you think?